### ISSUE TITLE
Weather

### ISSUE THREAD

**Issue by rpoovey** [AlecSchneider/nerdbar.widget#1]
Thanks for the awesome bar. 

I can't seem to get the weather widget to work though. All I get is "--". Never updates.

**Comment by AlecSchneider** [comment]
Hey, not super sure what the issue might be here :/ I didn't write the code for that widget myself. Sometimes I have the same bug but usually, it goes away quickly. I think it means it can't reach the API endpoint or figure out your location. You obviously need:
- internet
- have location services enabled in your Macs privacy setting.

Hope that helps :)

**Comment by rpoovey** [comment]
Yeah, I assumed that was the problems. Both of those things are verified on my end. Been a couple days and it hasn't cleared up. Is it possible the API is being called too much? Any pointers on how to create my own with Yahoo? Maybe that will resolve it.
